lib/friendly_shipping/services/tforce_freight/generate_create_bol_request_hash.rb

```diff
@@ -68,15 +68,15 @@ def payment(location, options)
                 name: location.company_name.presence || location.name,
                 contact: location.name,
                 email: location.email,
-                phone: { number: location.phone, },
+                phone: { number: location.phone }.compact,
```

Comment:
Will TForce accept an empty hash as the phone value?

Reply:
Wow, it sounds like they are following the bad habits of UPS.

Reply:
Apparently!